### DISCUSSION TITLE
Awesome tool. Is there anything like this to backup Google Drive?

### DISCUSSION THREAD

**Discussion by ilium007** [GAM-team/got-your-back#490] [General]
Looking for something that can back up a Google Drive for users in our GooglE Workspace account using a domain wide service account.

**Comment by SidSHaresSoftware** [comment]
Have you found a solution? If not, I suggest you look into the **SysTools [Google Workspace Backup](https://www.systoolsgroup.com/google-workspace-backup.html)** utility. It is the perfect solution for your needs. Fully GUI, no scripts or code, just click and save all Google Drive user data offline.

**Comment by JohnVillalovos** [comment]
I use rclone https://rclone.org/

**Reply by JohnVillalovos** [reply]
Another option is buy a Synology NAS and they have a utility to backup Google workspaces.
